web/packages/design/src/Label/Label.tsx

```diff
@@ -59,7 +59,12 @@ const kind = ({ kind, theme }: { kind?: LabelKind; theme: Theme }) => {
   };
 };
 
-type LabelKind = 'primary' | 'secondary' | 'warning' | 'danger' | 'success';
+export type LabelKind =
```

Comment:
Can we re-export this type from `design/src/Label/index.ts`? (you will only need to convert it to TS first)

This will allow us to import both the component and the type from the same module, e.g:
```ts
import Label, { LabelKind } from 'design/Label;
```